codeexamples/SYCLUsingCUDAStaticLibraryOfKernels/StaticLibraryBuildCUDAKernels/Makefile

```diff
@@ -0,0 +1,15 @@
+# Library names
+SRC_FILE ?= library.cu
+DST_FILE ?= libStaticLibOfFnsCUDAKernelsDXTC
+
+# Location of the CUDA Toolkit
+CUDA_PATH ?= /usr/local/cuda/bin
+
+# Target rules
+all: build
+
+build:
+	$(MAKE) -C bin all
+
+	
```

Comment:
There's some whitespace here we can remove

Reply:
White space removed from the file. Done.